docs/DeveloperGuide.md

```diff
@@ -13,8 +13,19 @@
 
 ## **Acknowledgements**
 
-WedLinker is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
-
+WedLinker is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org). <br>
+The following java classes in the directory `tp\src\main\java\seedu\address\model\task` were taken from one of our team member's [CS2103T Individual Project (IP)](https://github.com/DanzaSeah/tp/tree/master/src/main/java/seedu/address/model/task). These classes were modified to align with the requirements of the project
+- Task, Todo, Deadline, Event <br>
```

Comment:
This all looks good! I think might just also have to update the classes themselves that are affected, as per the policy on code reuse [found here](https://nus-cs2103-ay2425s1.github.io/website/schedule/week12/admin.html)

<img width="794" alt="image" src="https://github.com/user-attachments/assets/05c6ff31-7239-42d1-a302-c68aebde8de7">
<img width="783" alt="Screenshot 2024-11-12 at 3 04 42 AM" src="https://github.com/user-attachments/assets/b3542726-2863-49ce-9746-7f87564f7c15">

Reply:
oh, i will add that thanks